lib/models/blueprint.js

```diff
@@ -553,8 +557,14 @@ let Blueprint = CoreObject.extend({
       for (let i = 0; i < templateTagMatches.length; i++) {
         const match = templateTagMatches[i];
         const templateTag = substringBytes(code, match.range.start, match.range.end);
-        transformedWithTemplateTag = transformedWithTemplateTag.replace(templateTagIdentifier(i), templateTag);
-        transformedWithTemplateTag = transformedWithTemplateTag.replace(templateTagIdentifierBraces(i), templateTag);
+        if (match.type === 'class-member') {
+          transformedWithTemplateTag = transformedWithTemplateTag.replace(replacementClassMember(i), templateTag);
+        } else {
+          // babel-remove-types uses prettier under the hood, and adds trailing `;` where allowed,
```

Comment:
Seems we can also disable this, so that might be the better option:

```ts
// Remove types
const transformed = await removeTypes(strippedCode, false);
```

`false` === disable prettier.

Reply:
Comes with other issues, so ignoring this for now.